### PR TITLE
Rollups overview page

### LINKED ISSUE #686: Add Rollups overview page

### PR DESCRIPTION
**Pull request by Mackenzie-OO7**
## Overview

Resolves #686 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/celestia-app-commands.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ specs](https://github.com/celestiaorg/celestia-app/blob/main/x/paramfilter/READM
 Viewing the available proposals can be done with the query command:
 
 ```sh
-celestia-appd q gov proposals 
+celestia-appd q gov proposals
```
Comment (jcstein) [inline docs/nodes/celestia-app-commands.md:163]: OK. but out of scope for PR I think

File: docs/nodes/celestia-app-commands.md [hunk 2]
```diff
@@ -204,18 +204,21 @@ If we want to use the newer api, we can submit a proposal by first saving the
...
-            "@type":"/cosmos.params.v1beta1.ParameterChangeProposal",
-            "title":"title",
-            "description":"description",
-            "changes":[{"subspace":"staking","key":"MaxValidators","value":"103"}]
-            },
-            "initial_deposit":[{"denom":"utia","amount":"1000000000"}],
-            "proposer":"celestia10d07y265gmmuvt4z0w9aw880jnsr700jtgz4v7"
-        }
-    ]
+  "messages": [
```
Comment (jcstein) [inline docs/nodes/celestia-app-commands.md:207]: out of scope for PR

File: sidebars.js [hunk 3]
```diff
@@ -361,7 +361,8 @@ const sidebars = {
       type: "category",
       label: "Deploy a Rollup",
 			link: {
-				type: "generated-index",
+        id: "developers/rollup-overview",
```
Comment (jcstein) [inline sidebars.js:364]: this should match the formatting for the rest of `sidebars.js` for this addition

File: docs/developers/rollup-overview.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+sidebar_label: Deploy a Rollup
+description: Learn how to deploy rollups on Celestia.
+---
+
+# Rollups
+
+This section covers the different options for deploying rollups on Celestia.
+
+It contains guides and tutorials on how to deploy rollups on Celestia git add the provided options.
```
Comment (jcstein) [inline docs/developers/rollup-overview.md:10]: "git add the provided options"? what does this mean?

File: docs/developers/rollup-overview.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    - [Deploy an OP Stack devnet](./optimism-devnet.mdx)
+    - [Deploy an OP Stack testnet on Celestia](./optimism.mdx)
+  - [Rollups as a Service](https://docs.celestia.org/category/rollups-as-a-service/)
+- Soverign SDK
+  - [How to create a zk-rollup and run a full node using Soverign SDK](https://github.com/Sovereign-Labs/sovereign-sdk/tree/main/examples/demo-rollup#demo-rollup)
+- IBC Relayers
+  - [How to establish IBC connections using the Hermes relayer](./ibc-relayer.md)
+- Full-stack modular blockchain development
+  - [How to build a full stack modular dapp with React, Vite, RainbowKit, Celestia and Foundry](./full-stack-modular-development-guide.md)
+- [Dymension](https://dymension.xyz/)
```
Comment (jcstein) [inline docs/developers/rollup-overview.md:34]:
```suggestion
- Build a [rollapp with Dymension's Roller](https://docs.dymension.xyz/build/roller)
```

**Review by jcstein — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 3 above)
Comment (Mackenzie-OO7) [reply sidebars.js:364]: Thank you for your review. I will implement the changes requested. Could you clarify on this part though:

> this should match the formatting for the rest of `sidebars.js` for this addition

**Comment by Mackenzie-OO7** [comment]
I've made the requested changes now.

**Review by jcstein — APPROVED** [review]
:shipit:

**Comment by jcstein** [comment]
cc @Mackenzie-OO7 for the GitPOAP!

**Comment by Mackenzie-OO7** [comment]
> cc @Mackenzie-OO7 for the GitPOAP! thank you for your contribution to the documentation.

Thank you!

**Pull request merged**